aiidalab/aiidalab-home
# Disable prompt dialog when exiting home page

This part of a solution for #133 (see more info there). 

In this PR, I am only affecting the home page. If this proves to work, we can apply the same for other pages, such as the appstore or the "Manage app" page.

Essentially, I just added a piece of Javascript that disables the prompt functionality, coming from Jupyter notebook. I've put the cell purposefully at the end of the notebook so that it is executed last. When a developer would be working on this notebook, the alert will not be disabled unless they execute the last cell.

Testing plan:
1. Open home page
2. Open "Tasks". You should see the running home page notebook.
3. Close the home page. There should be no prompt warning from the browser.
4. Verify that the notebook disappears from the task list.

@unkcpz Can you give it a try? It would be great to have this merge for the next release so we can test it out more broadly.
